lnbits/settings.py

```diff
@@ -290,6 +291,8 @@ class EnvSettings(LNbitsSettings):
     lnbits_extensions_path: str = Field(default="lnbits")
     super_user: str = Field(default="")
     version: str = Field(default="0.0.0")
+    # new cache version each time you restart the server
+    cache_version: int = Field(default=time())
```

Comment:
Neat trick!

Reply:
- this means the client will re-download the files even if nothing has changed in the codebase (only as server restart took place). A bit sub-optimal.

Reply:
- this field (if it will be kept) should have a name that represents its value:
```suggestion
    server_startup_time: int = Field(default=time())
```
- this value could then also be used to for example to show the uptime of the server (on the admin page)

Reply:
> * this means the client will re-download the files even if nothing has changed in the codebase (only as server restart took place). A bit sub-optimal.

i agree, that is my concern aswell. but i think the tradeoff is worth it. you should try to minimize restarting to only after updates.